elytron/src/test/java/org/wildfly/extension/elytron/CertificateAuthoritiesTestCase.java

```diff
@@ -263,7 +222,8 @@ public void testCreateAccountWithEmptyStagingUrlAndStagingValueTrue() throws Exc
             operation.get(ClientConstants.OP).set(ElytronDescriptionConstants.CREATE_ACCOUNT);
             operation.get(ElytronDescriptionConstants.AGREE_TO_TERMS_OF_SERVICE).set(true);
             operation.get(ElytronDescriptionConstants.STAGING).set(true);
-            assertFailed(services.executeOperation(operation));
```

Comment:
I think we should keep this assertion to make sure the operation really did fail before attempting to get the failure description.

Reply:
Fixed for all